### ISSUE TITLE
Corax indexing error: Got an invalid container id: -1

### ISSUE THREAD

**Issue by loraderon** [ravendb/ravendb#18238]
Using RavenDB Cloud we got an error in one of our Corax indexes. It worked earlier. I changed it to Lucene and it now works again.

```
Unexpected exception occurred: System.InvalidOperationException: Got an invalid container id: -1
   at Voron.Data.Containers.Container.GetMutable(LowLevelTransaction llt, Int64 id) in D:\Builds\RavenDB-Stable-6.0\60012\src\Voron\Data\Containers\Container.cs:line 1019
   at Corax.Indexing.IndexWriter.AddEntriesToTerm(Span`1 tmpBuf, Int64 idInTree, Boolean isNullTerm, EntriesModifications& entries, Int64& termId) in D:\Builds\RavenDB-Stable-6.0\60012\src\Corax\Indexing\IndexWriter.cs:line 1493
   at Corax.Indexing.IndexWriter.InsertNumericFieldDoubles(Tree entriesToTermsTree, IndexedField indexedField, Span`1 tmpBuf) in D:\Builds\RavenDB-Stable-6.0\60012\src\Corax\Indexing\IndexWriter.cs:line 1795
   at Corax.Indexing.IndexWriter.Commit[TStatsScope](TStatsScope stats) in D:\Builds\RavenDB-Stable-6.0\60012\src\Corax\Indexing\IndexWriter.cs:line 863
   at Raven.Server.Documents.Indexes.Persistence.Corax.CoraxIndexWriteOperation.Commit(IndexingStatsScope stats) in D:\Builds\RavenDB-Stable-6.0\60012\src\Raven.Server\Documents\Indexes\Persistence\Corax\CoraxIndexWriteOperation.cs:line 79
   at Raven.Server.Documents.Indexes.Index.DoIndexingWork(IndexingStatsScope stats, CancellationToken cancellationToken) in D:\Builds\RavenDB-Stable-6.0\60012\src\Raven.Server\Documents\Indexes\Index.cs:line 2478
   at Raven.Server.Documents.Indexes.Index.ExecuteIndexing() in D:\Builds\RavenDB-Stable-6.0\60012\src\Raven.Server\Documents\Indexes\Index.cs:line 1682
```

**Comment by arekpalinski** [comment]
@loraderon we are aware of that and have a lead already.

**Comment by maciejaszyk** [comment]
@loraderon I created a test and issue for this.
You can track the progress here: https://issues.hibernatingrhinos.com/issue/RavenDB-22141

**Comment by arekpalinski** [comment]
This got fixed in https://github.com/ravendb/ravendb/pull/18240 and it will be released in stable 6.0 build next week.

**Comment by loraderon** [comment]
> This got fixed in #18240 and it will be released in stable 6.0 build next week.

Thank you! 

Will the cloud instances also be updated next week?

**Comment by arekpalinski** [comment]
Please contact us via the support form in the cloud portal and we'll update your instance right after publishing the new build.

**Comment by arekpalinski** [comment]
This is fixed in 6.0.101. Please contact us via RavenDB Cloud portal and we'll schedule the update of your product. Please note that the reset of the affected index will be necessary after the upgrade.
